views/404.pug

```diff
@@ -0,0 +1,27 @@
+extends layout.pug
+
+block content
+    h2.text-center.mb-4 Page Not Found
+
+    .wrap.text-center.pb-2
+        p
+            | We couldn’t find what you’re looking for on 
+            em #{siteUrl}
+            | .
+        p
+            a.btn.btn-lg.btn-success(href=siteUrl) Take Me To The Homepage
+
+    .mt-5
+        .row
+            .col-md-6
+                h3 What happened?
+                p.lead
+                    | A 404 error implies that the file or page that you’re looking for
+                    | could not be found.
+            .col-md-6
+                h3 What can I do?
+                p.lead
+                    | Please use your browser’s back button and check that you’re in the right place.
+                    | If you need immediate assistance, please send us an email instead.
```

Comment:
Maybe I should just link to our GitHub issues instead of saying send us an email?